tools/gen-wire-tests/juju.config

```diff
@@ -78,6 +81,7 @@ folders:
     - branches
     - charmhub
     - cli
+    - constraints
```

Comment:
i think having a constraints test on microk8s is something we'd want

Reply:
I need to check what constraints we are supporting for microk8s... and, firstly add them to the constraints test suite

Reply:
Constrains tests for microk8s are not implemented yet. So we should leave this 'skip'